indico/modules/events/abstracts/templates/management/_notification_tpl_list.html

```diff
@@ -1,6 +1,6 @@
 {% from 'message_box.html' import message_box %}
 
-{% macro render_notification_list(event) -%}
+{% macro render_notification_list(event, track, states, contr_type, email_tpl) -%}
```

Comment:
It's not a single track but a dictionary of tracks, right? I would name it `tracks`, if that's the case. The same applies for `contrib_type` and `email_tpl`, I believe.